packages/mds-metrics-sheet/package.json

```diff
@@ -7,12 +7,7 @@
   "files": [
     "dist/"
   ],
-  "scripts": {
-    "build": "tsc --build tsconfig.build.json",
-    "test": "yarn test:eslint && yarn test:unit",
-    "test:eslint": "eslint --ignore-path ../../.gitignore '**/*.ts'",
-    "test:unit": "DOTENV_CONFIG_PATH=../../.env nyc --lines 60  ts-mocha --project ../../tsconfig.json"
-  },
+  "scripts": {},
```

Comment:
This is temporary just so other packages can build?

Reply:
Basically, `mds-daily` and `mds-metrics-sheet` both need significant reworks to represent 1.0.0 data properly. There's a good chance that they will be deprecated by the time 1.0.0 hits production (or even sandbox), so updating them is pretty low on the priority bucket list.